pkg/venafi/tpp/connector.go

```diff
@@ -653,6 +653,14 @@ func (c *Connector) prepareRequest(req *certificate.Request, zone string) (tppRe
 	//    - true: Clear the Disabled attribute, reenable, and then renew the certificate (in this request). Reuse the same CertificateDN, that is also known as a Certificate object.
 	tppReq.Reenable = true
 
+	// If enable timeout is defined by the user in the request, we use it in order
+	// override API's timeout for the CA to finish issuance
+	if req.Timeout != 0 {
```

Comment:
Change it to `if req.Timeout > 0` to rule out negative values

Reply:
Done by latest commit